Build failure: mat2

### Nixpkgs version

- Unstable (25.11)

### Steps to reproduce

- build *mat2*

### Can Hydra reproduce this build failure?

Yes, Hydra can reproduce this build failure.

### Link to Hydra build job

https://hydra.nixos.org/build/305820043

### Relevant log output

```console
=================================== FAILURES ===================================
____________________________ TestCleaning.test_html ____________________________

self = <tests.test_libmat2.TestCleaning testMethod=test_html>

    def test_html(self):
        shutil.copy('./tests/data/dirty.html', './tests/data/clean.html')
        p = web.HTMLParser('./tests/data/clean.html')
    
        meta = p.get_meta()
        self.assertEqual(meta['author'], 'jvoisin')
    
        ret = p.remove_all()
        self.assertTrue(ret)
    
        p = web.HTMLParser('./tests/data/clean.cleaned.html')
        self.assertEqual(p.get_meta(), {})
        self.assertTrue(p.remove_all())
    
        os.remove('./tests/data/clean.html')
        os.remove('./tests/data/clean.cleaned.html')
        os.remove('./tests/data/clean.cleaned.cleaned.html')
    
        with open('./tests/data/clean.html', 'w') as f:
            f.write('<title><title><pouet/><meta/></title></title><test/>')
>       p = web.HTMLParser('./tests/data/clean.html')
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

tests/test_libmat2.py:633: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
libmat2/web.py:57: in __init__
    self.__parser.feed(f.read())
/nix/store/iyff8129iampdw13nlfqalzhxy8y1hi9-python3-3.13.6/lib/python3.13/html/parser.py:159: in feed
    self.goahead(0)
/nix/store/iyff8129iampdw13nlfqalzhxy8y1hi9-python3-3.13.6/lib/python3.13/html/parser.py:228: in goahead
    k = self.parse_endtag(i)
        ^^^^^^^^^^^^^^^^^^^^
/nix/store/iyff8129iampdw13nlfqalzhxy8y1hi9-python3-3.13.6/lib/python3.13/html/parser.py:472: in parse_endtag
    self.handle_endtag(tag)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

self = <libmat2.web._HTMLParser object at 0x7ffff4ac0f30>, tag = 'title'

    def handle_endtag(self, tag: str):
        if not self.__validation_queue:
>           raise ValueError("The closing tag %s doesn't have a corresponding "
                             "opening one in %s." % (tag, self.filename))
E           ValueError: The closing tag title doesn't have a corresponding opening one in ./tests/data/clean.html.

libmat2/web.py:133: ValueError
```

### Additional context

_No response_

### System metadata

 - system: `"x86_64-linux"`
 - host os: `Linux 6.12.43, NixOS, 25.11 (Xantusia), 25.11.20250822.f937f8e`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Lix, like Nix) 2.93.3
System type: x86_64-linux
Additional system types: i686-linux, x86_64-v1-linux, x86_64-v2-linux, x86_64-v3-linux
Features: gc, signed-caches
System configuration file: /etc/nix/nix.conf
User configuration files: /home/misile/.config/nix/nix.conf:/etc/xdg/nix/nix.conf:/home/misile/.nix-profile/etc/xdg/nix/nix.conf:/nix/profile/etc/xdg/nix/nix.conf:/home/misile/.local/state/nix/profile/etc/xdg/nix/nix.conf:/etc/profiles/per-user/misile/etc/xdg/nix/nix.conf:/nix/var/nix/profiles/default/etc/xdg/nix/nix.conf:/run/current-system/sw/etc/xdg/nix/nix.conf
Store directory: /nix/store
State directory: /nix/var/nix
Data directory: /nix/store/s7izz6akz2q6wilmbll41nrwnfrgfl68-lix-2.93.3/share`
 - channels(root): `"nixos"`
 - nixpkgs: `/nix/store/bs1sz392mx4ig6l17a6baj8qv1w9pr4s-source`

### Notify maintainers

@dotlambda

---

**Note for maintainers:** Please tag this issue in your pull request description. (i.e. `Resolves #ISSUE`.)


### I assert that this issue is relevant for Nixpkgs

- [x] I assert that this is a bug and not a support request.
- [x] I assert that this is not a [duplicate of an existing issue](https://github.com/NixOS/nixpkgs/issues?q=is%3Aopen+is%3Aissue+label%3A%220.kind%3A+build+failure%22). 
- [x] I assert that I have read the [NixOS Code of Conduct](https://github.com/NixOS/.github/blob/master/CODE_OF_CONDUCT.md) and agree to abide by it.

### Is this issue important to you?

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

--- a/pkgs/by-name/ma/mat2/package.nix
+++ b/pkgs/by-name/ma/mat2/package.nix
@@ -1,0 +1,9 @@
+{
+  # On Python 3.13, `tests/test_libmat2.py::TestCleaning::test_html` fails with
+  #
+  #     ValueError: The closing tag title doesn't have a corresponding opening one in ./tests/data/clean.html.
+  python312Packages,
+}:
+
+with python312Packages;
+toPythonApplication mat2

--- a/pkgs/top-level/all-packages.nix
+++ b/pkgs/top-level/all-packages.nix
@@ -2228,8 +2228,6 @@ with pkgs;
 
   maliit-keyboard = libsForQt5.callPackage ../applications/misc/maliit-keyboard { };
 
-  mat2 = with python3.pkgs; toPythonApplication mat2;
-
   materialx = with python3Packages; toPythonApplication materialx;
 
   # while building documentation meson may want to run binaries for host

